test-new/EmailNrGuardianVerifier/EmailNrGuardianVerifierBase.t.sol

```diff
@@ -0,0 +1,460 @@
+    ) public {
+        if (getCommandHandlerType() == commandHandlerType) {
+            vm.skip(true);
+        } else {
+            vm.skip(false);
+        }
+    }
+
+    // Helper functions
+    function computeGuardianVerifierAuthAddress(
+        address guardianVerifierImplementation,
+        address account,
+        bytes32 accountSalt,
+        bytes memory verifierInitData
+    ) public view override returns (address) {
+        return
+            emailRecoveryModule.computeGuardianVerifierAddress(
+                guardianVerifierImplementation,
+                account,
+                accountSalt,
+                verifierInitData
+            );
+    }
+
+    // Helper functions
+    function deployModule() public override {
+        // Deploy the email recovery factory after removing email guardian verifier related logic
+        emailRecoveryFactory = new EmailRecoveryFactory();
+
+        bytes32 recoveryModuleSalt = bytes32(uint256(0));
+        emailRecoveryModuleAddress = emailRecoveryFactory
+            .deployEmailRecoveryModule(
+                recoveryModuleSalt,
+                minimumDelay,
+                killSwitchAuthorizer,
+                validatorAddress,
+                functionSelector
+            );
+        emailRecoveryModule = EmailRecoveryModule(emailRecoveryModuleAddress);
+
+        if (
+            getCommandHandlerType() ==
+            CommandHandlerType.AccountHidingRecoveryCommandHandler
+        ) {
+            AccountHidingRecoveryCommandHandler(commandHandlerAddress)
+                .storeAccountHash(accountAddress1);
+            AccountHidingRecoveryCommandHandler(commandHandlerAddress)
+                .storeAccountHash(accountAddress2);
+            AccountHidingRecoveryCommandHandler(commandHandlerAddress)
+                .storeAccountHash(accountAddress3);
+        }
+    }
+
+    // Helper functions
+    function setRecoveryData() public override {
+        functionSelector = bytes4(keccak256(bytes("changeOwner(address)")));
+        recoveryCalldata = abi.encodeWithSelector(functionSelector, newOwner1);
+        recoveryData = abi.encode(validatorAddress, recoveryCalldata);
+        recoveryDataHash = keccak256(recoveryData);
+    }
+
+    // bytes32 pubkey;
+    // bytes32 nullifier;
+    struct NoirProof {
+        bytes proof;
+    }
+
+    //NOTE: This is a mock proof, it isn't present here now
+    function generateEmailNrProof(
+        bytes32 nullifier,
+        bytes32 accountSalt
+    ) public view returns (EmailProof memory) {
+        EmailProof memory emailProof;
+        emailProof.domainName = "gmail.com";
+        emailProof.timestamp = block.timestamp;
+        emailProof.maskedCommand = "";
+        emailProof.accountSalt = accountSalt;
+        emailProof.isCodeExist = true;
+
+        string memory proofFile = vm.readFile(
+            string.concat(
+                vm.projectRoot(),
+                "/test-new/EmailNrGuardianVerifier/proof.json"
```

Comment:
Which file is this I can't see it?

Reply:
Oh this was removed, this whole function is no longer needed.